environments/development/common/cognito.tf

```diff
@@ -19,3 +32,18 @@ resource "aws_route53_record" "cognito_custom_domain" {
     zone_id                = module.cognito.cloudfront_distribution_zone_id
   }
 }
+
+resource "aws_ssm_parameter" "cognito_public_keys" {
+  name        = "/COGNITO_PUBLIC_KEYS_URL"
```

Comment:
Do we want to prefix this with the environment?

Reply:
![image](https://github.com/trade-tariff/trade-tariff-platform-aws-terraform/assets/8156884/6ed49264-9364-44fa-b0b2-086a664cd3e0)

Just to be consistent not because I have a massive preference

Reply:
Actually, probably `/${var.environment}/FPO_DEVELOPER_HUB_COGNITO_PUBLIC_USER_URL` :man_shrugging: